charts/geonode/README.md

```diff
@@ -147,7 +147,7 @@ Helm Chart for Geonode. Supported versions: Geonode: 4.1.3, Geoserver: 2.23.0, p
 | geoserver.resources.limits.memory | string | `"4Gi"` | limits memory as in resource.limits.memory (https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) |
 | geoserver.resources.requests.cpu | int | `1` | requested cpu as in resource.requests.cpu (https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) |
 | geoserver.resources.requests.memory | string | `"1Gi"` | requested memory as in resource.requests.memory (https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) |
-| geoserver.secret.admin_password | string | `"geoserver"` | geoserver admin password |
+| geoserver.secret.admin_password | string | `"geoserver"` | geoserver admin password only gets only changed when the previous username/password combination is the default one (admin/geoserver) |
```

Comment:
To be picky here: there is also `GEOSERVER_FACTORY_PASSWORD`. IMO, the naming could be better here, because this setting can be seen (more or less) as currently set password [when you want to set a new GeoServer admin password](https://github.com/GeoNode/geonode-docker/blob/05cf135cf4a01889eabe6804cf70853e9c77ada3/docker/geoserver/tasks.py#L44).

If we want to limit this configuration to-be-set-once (as documented) I am fine with it, though.

Reply:
This thing got a new spin now. The password now gets changed by the Geserver `entrypoint.sh`, which wasn't working before because of the missing envvar: `FORCE_REINIT`. Therefor i removed the code to set the password from the `geonode-k8s/tasks.py`.

Further I added the factory_admin_password variable and gave some documentation on how to use it

Reply:
Do you want to stick with the factory_ part? To me it is actually the currently set password (initial or not) which can be used to set a new password. However, it is also very valid to keep naming in sync with the upstream.